examples/fscloud/main.tf

```diff
@@ -62,6 +62,7 @@ module "event_streams" {
   tags                       = var.resource_tags
   topics                     = var.topics
   existing_kms_instance_guid = var.existing_kms_instance_guid
+  metrics                    = []
```

Comment:
can you add the allowed values here so they get tested in the tests we run?

Reply:
`examples/fscloud` doesn't set the plan to `enterprise-3nodes-2tb`, so is using a standard plan, and these options are not available for standard.  Checking our provision code, I see that they're just ignored, and don't cause an error - but I can't promise it will always work that way. 

So I could include the allowed values, but it would only be a test of the terraform validator, and might break in the future if we improve the parameters checking.

Reply:
`examples/fscloud` doesn't set the plan to `enterprise-3nodes-2tb`, so is using a standard plan, and these options are not available for standard.  Checking our provision code, I see that they're just ignored, and don't cause an error - but I can't promise it will always work that way. 

So I could include the allowed values, but it would only be a test of the terraform validator, and might break in the future if we improve the parameters checking.

Reply:
The fscloud example is calling the `modules/fscloud` submodule, and this is hard code to use the `"enterprise-3nodes-2tb"` plan